app/main.py

```diff
@@ -1,8 +1,61 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) \
+            -> None:
+        if not (1 <= comfort_class <= 7):
+            raise ValueError(
+                "The value of `comfort_class` must be in the range from 1 to 7"
+            )
+        self.comfort_class = comfort_class
+        if not (1 <= clean_mark <= 10):
+            raise ValueError(
+                "The value of `clean_mark` must be in the range from 1 to 10"
+            )
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        if not (1.0 <= distance_from_city_center <= 10.0):
+            raise ValueError(
+                "The value of `distance_from_city_center` "
+                "must be in the range from 1.0 to 10.0"
+            )
+        self.distance_from_city_center = distance_from_city_center
+        if not (1 <= clean_power <= 10):
+            raise ValueError(
+                "The value of `clean_power` must be in the range from "
+                "1 to 10"
+            )
+        self.clean_power = clean_power
+        if not (1.0 <= average_rating <= 5.0):
+            raise ValueError(
+                "The value of `average_rating` must be in the range from "
+                "1.0 to 5.0"
+            )
```

Comment:
Yea, you can do whatever you want in init. It's always better to do validation before you create some object with incorrect data. Or you can create a separate method with some parameters and call it inside init method.
Do you see how your code became harder to read? Also note that all if statements have a common pattern and it means you can create some method in order to not copy paste almost the same code